.gitattributes

```diff
@@ -1,6 +1,6 @@
 * text=auto
 
-**/Generated/**/*.cs linguist-generated=true
-**/Generated/**/*.csproj linguist-generated=true
+**/Generated/*.cs linguist-generated=true
+**/*.csproj linguist-generated=true
```

Comment:
This definition is too generic it would include csproject that are not generated.

We might want to have multiple definitions that are prefixed with directories we know contain generated code like samples, tests/TestServerProjects etc.